Tests/differ.py

```diff
@@ -163,6 +163,7 @@ def _get_all_file_paths_in_dir_tree(start_path):
 
         # Make the paths relative
         all_paths = [path.replace(start_path, '') for path in all_paths]
+        all_paths.sort()
```

Comment:
I don't think this is necessary because the paths are not going to be compared between Win and Mac

Reply:
Sorry, I made a mistake in the comment - the difference is between the reference directory tree and the new temporary director tree when the test is run under Windows, not Mac and Windows.

Reply:
OK, then use `sorted()` instead.